content/menu/index.md

```diff
@@ -14,7 +14,7 @@ headless: true
   * [Block Validation]({{< relref "/docs/validation.md" >}})
 * [Network]({{< relref "/docs/networking.md" >}})
   * [Data Propagation]({{< relref "/docs/data-propagation.md" >}})
-  * [Custom Protocols]({{< relref "/docs/network-protocols.md" >}})
+  * [FC specific Protocols]({{< relref "/docs/network-protocols.md" >}})
```

Comment:
FC -> Filecoin? I don't think we abbreviate it anywhere else.

Reply:
Yeah,  I wasn't sure about this one. We dropped "Custom" in the header and called it "Filecoin specific", but in the sidebar this gets to long. So maybe dropping the reference?
```suggestion
  * [Specific Protocols]({{< relref "/docs/network-protocols.md" >}})
```

But that seems weird somehow. So maybe rather?

```suggestion
  * [Special Protocols]({{< relref "/docs/network-protocols.md" >}})
```

Reply:
I think Specific works well enough